File: 2021년/SW Expert Academy/D1/Python/2027.py
# 다음의 텍스트를 그대로 출력하라
# #++++
# +#+++
# ++#++
# +++#+
# ++++#

for row in range (5):
    for column in range (5):
        if row == column:
            print("#", end="")
        else:
            print("+", end="")
    print()

File: 2021년/SW Expert Academy/D1/C/2027.c
// 주어진 텍스트를 그대로 출력하라
// #++++
// +#+++
// ++#++
// +++#+
// ++++#

#include <stdio.h>
#pragma warning(disable:4996)

main()
{
	int row, column, i;

	for (row = 0; row < 5; row++)
	{
		for (column = 0; column < 5; column++)
		{
			if (row == column)
				printf("#");
			else
				printf("+");
		}
		printf("\n");
	}

}